.github/workflows/building.yml

```diff
@@ -0,0 +1,17 @@
+name: Building tests
+on: [push, pull_request]
+
+jobs:
+  test_applications:
+    runs-on: ubuntu-latest
+    container: 
+      image: ghcr.io/juansapriza/xheep-compiler:riscv-clang
```

Comment:
we should be having the image using the lab user rather than a personal one

Reply:
@JuanSapriza any news about this?

Reply:
not yet :(